src/utilities/codemirror/satf/satf.test.ts

```diff
@@ -0,0 +1,853 @@
+import { testTree } from '@lezer/generator/dist/test';
+import { expect, test } from 'vitest';
+import { SatfLanguage } from './satf';
+
+test.each([
```

Comment:
Can these cases be organized into an array of objects with keys describing their purpose to make the test a little more grokable?

Reply:
I did a minor update so you can see which test is running 

✓ src/utilities/codemirror/satf/satf.test.ts (18)
   ✓ grammar tests - parse tree structure (18)
     ✓ Empty SATF
     ✓ Simple SATF
     ✓ Header Gone
     ✓ Empty SeqHeader Gone
     ✓ Multiple Activity Groups
     ✓ Activity Group optional values
     ✓ Activity Group inline optional values
     ✓ Parameters
     ✓ Variables
     ✓ Variables and Parameters
     ✓ Commands
     ✓ Dynamic Commands
     ✓ Notes
     ✓ Activate
     ✓ Loops
     ✓ Assignment
     ✓ Ground Event
     ✓ sample satf